### PR TITLE
Ignore dotfiles

### PR DESCRIPTION
**Pull request by BenPope**
Fixes: https://github.com/lyft/goruntime/issues/15

### CODE REVIEW

File: loader/loader_test.go [hunk 1]
```diff
@@ -53,6 +53,8 @@ func TestSymlinkRefresher(t *testing.T) {
 	makeFileInDir(assert, tempDir+"/testdir1/app/file1", "hello")
 	makeFileInDir(assert, tempDir+"/testdir1/app/dir/file2", "world")
 	makeFileInDir(assert, tempDir+"/testdir1/app/dir2/file3", "\n 34  ")
+	makeFileInDir(assert, tempDir+"/testdir1/app/dir3/.file4", ".file4")
+	makeFileInDir(assert, tempDir+"/testdir1/app/.dir/file5", ".dir")
```
Comment (topliceanu) [inline loader/loader_test.go:57]: I assume these paths get normalised to remove things like `foo/./bar` or  `foo/../bar`?

Comment (BenPope) [reply loader/loader_test.go:57]: I doubt https://golang.org/pkg/path/filepath/#Walk would create such paths.

Comment (lemccarthy) [reply loader/loader_test.go]: I've run into the same issue, and would appreciate the PR being merged. I'm close to having a working helm implementation of this service.

**Comment by BenPope** [comment]
Upstream: https://github.com/lyft/goruntime/pull/17

**Pull request closed without merging**